teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './person'
+
+# Teacher class definition
+class Teacher < Person
+  def initialize(specialization)
+    @specialization = specialization
+    super(age, name, parent_permission: parent_permission)
+  end
```

Comment:
- [ ] Kindly add `age`, `name`, and `parent_permission` as a parameter for a constructor so you can pass it to super.
- [ ] Kindly call the super before `@specialization = specialization` so it will be like
```
   super(age, name, parent_permission: parent_permission)
   @specialization = specialization
```